wallet-new/test/Integration/Util.hs

```diff
@@ -0,0 +1,20 @@
+module Integration.Util where
+
+import           Universum
+
+import           Cardano.Wallet.Client.Http (ClientError, WalletResponse (..))
+
+
+type Resp' a = Either ClientError (WalletResponse a)
+
+runSpec :: r -> ReaderT r IO () -> IO ()
+runSpec =
+    flip runReaderT
```

Comment:
There's no need to prematurely make this a `()` type, or in `IO`. I'd rather see this fully general so it can be more useful.

Reply:
:+1: Indeed.